tests/main/lxd/task.yaml

```diff
@@ -7,7 +7,8 @@ backends: [-autopkgtest]
 # Only run this on ubuntu 16+, lxd will not work on !ubuntu systems
 # currently nor on ubuntu 14.04
 # TODO: enable for ubuntu-21.10+ once the lxd image is published
-systems: [ubuntu-16*, ubuntu-18*, ubuntu-20*, ubuntu-21.04-*, ubuntu-core-*]
+# ubuntu-18.04-32: i386 is not supported by lxd
+systems: [ubuntu-16*, ubuntu-18.04-64, ubuntu-20*, ubuntu-21.04-*, ubuntu-core-*]
```

Comment:
should we also (maybe not as part of this PR, but generally) add `ubuntu-21.10-*` too ?

Reply:
ah okay no problem them